packages/block-library/src/gallery/edit.js

```diff
@@ -440,7 +440,7 @@ function GalleryEdit( props ) {
 		return <View { ...blockProps }>{ mediaPlaceholder }</View>;
 	}
 
-	const shouldShowSizeOptions = ! isEmpty( imageSizeOptions );
```

Comment:
It turns out that some images have no size details, particularly those that have been cropped to max, so need to account for this so loader doesn't show indefinitely